Quickstart/azure-search-quickstart/azure-search-quickstart/Address.Methods.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace azure_search_quickstart
+{
+    using System;
+    using System.Text;
+    using Newtonsoft.Json;
+
+    public partial class Address
+    {
+        // This implementation of ToString() is only for the purposes of the sample console application.
+        // You can override ToString() in your own model class if you want, but you don't need to in order
+        // to use the Azure Search .NET SDK.
+
+        public override string ToString()
+        {
+            var builder = new StringBuilder();
```

Comment:
This is simple enough that using `StringBuilder` is not worth the effort. It could be written much more simply using interpolated strings, like this:

```csharp
public override string ToString() =>
    IsEmpty ?
        string.Empty :
        $"{StreetAddress}\n{City}, {StateProvince} {PostalCode}\n{Country}";
```